Apps/AdminWebClient/src/Server/Constants/UserQueryType.cs

```diff
@@ -39,5 +39,10 @@ public enum UserQueryType
         /// Query by Health-Directed identifier.
         /// </summary>
         Hdid,
+
+        /// <summary>
+        /// Query by dependent Personal Health Number.
+        /// </summary>
+        Dependent,
```

Comment:
Nit: this whole enum can be deleted now that AdminWebClient is now using the UserQueryType in CommonData instead of this one.